articles/how-to-create-a-dataset-for-machine-learning/index.md

```diff
@@ -0,0 +1,108 @@
+
+2. [ActiveClean](https://activeclean.github.io/) 
+
+ActiveClean is an iterative cleaning framework that cleans samples of data based on how much cleaning improves the models' accuracy. This means that you only need to clean a small subset of the data to achieve a model similar to a fully cleaned dataset.
+
+3. [BoostClean](https://arxiv.org/pdf/1711.01299.pdf) 
+
+BoostClean automatically detects and repairs errors in data using statistical boosting. Statistical boosting enables the system to find the best ensemble of pairs that maximize the final model's accuracy.
+
+4. [MLClean](https://arxiv.org/pdf/1904.10761.pdf)
+
+MLClean is the most recent data cleaning framework. The framework performs three main tasks: data sanitization, traditional data cleaning, and unfairness mitigation in data. It also integrates artificial intelligence to clean data to achieve robust, accurate, and fair models.     
+
+An important point to note is that you shouldn't clean too much. Ideally, cleaning a dataset should not result in a dataset that's no longer representative of the population that you are looking to perform a study on.
+
+### Data Labeling
+
+Now you have acquired enough data to have a representative dataset, cleaned, and in the right format. 
+
+Time to label that data? Maybe.
+
+The answer to that question solely depends on whether you are using supervised learning or unsupervised learning. Unsupervised learning doesn't require your data to be labeled, while supervised learning does require data labeling.
+
+The processes of labeling can be subjective and also labor-intensive. One can utilize crowdsourcing platforms such as [Amazon Mechanical Turk (MTurk) ](https://www.mturk.com/) and [Citizen Science](https://en.wikipedia.org/wiki/Citizen_science) to achieve this goal.  
+
+### Wrapping Up
+
+Is that the end of the process for creating a dataset? Well, probably, probably not.
+
+Training a model can reveal some issues that may negatively impact the outcomes that you may try to predict or classify. Often, these issues can be traced back to the dataset itself. You may have to return to the data acquisition, data cleaning, and data labeling drawing board to figure out how to create a better dataset for your ML system. But if your dataset lacks any issues, you're good to go.
+
+That's it for this article on how to create a dataset for machine learning. Hope the article was helpful.
+
+### References
+
+1. [A Survey on Data Collection for Machine Learning](https://arxiv.org/pdf/1811.03402.pdf)  
+2. [YouTube-8M Segments Dataset](https://research.google.com/youtube8m/)
+3. [Amazon Mechanical Turk (MTurk) ](https://www.mturk.com/)
+4. [Citizen Science](https://en.wikipedia.org/wiki/Citizen_science)
+5. [ImageNet project](http://www.image-net.org/) 
+6. [HoloClean](http://www.holoclean.io/) 
+7. [BoostClean](https://arxiv.org/pdf/1711.01299.pdf) 
+8. [MLClean](https://arxiv.org/pdf/1904.10761.pdf)
```

Comment:
```suggestion
8. [MLClean](https://arxiv.org/pdf/1904.10761.pdf)

---
Peer Review Contributions by: [Srishilesh P S](/engineering-education/authors/srishilesh-p-s/)
```

Reply:
I don't understand what the correction is here

Reply:
@Bayler This is not a correction. I have added PRC (Peer Reviewers Contribution) line. As a new peer-reviewer, I don't have permissions to commit, so I added it as a review.

Reply:
Oh. It's okay. Thanks for the explanation.